src/main/java/com/beerjournal/BeanConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.beerjournal;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.beerjournal.datamodel.repository.CollectableObjectsRepository;
+import com.beerjournal.datamodel.repository.UserCollectionRepository;
+import com.beerjournal.datamodel.repository.mongo.CollectableObjectsRepositoryImpl;
+import com.beerjournal.datamodel.repository.mongo.UserCollectionRepositoryImpl;
+
+@Configuration
```

Comment:
Wydzieliłem to do osobne konfiguracji aby mieć jakieś centralne miejsce do sterowania autowire, wprzypadku np podmiany implementacji repozytoriów (i samej bazy)